packages/flutter/lib/src/widgets/layout_builder.dart

```diff
@@ -244,6 +244,61 @@ mixin RenderConstrainedLayoutBuilder<ConstraintType extends Constraints, ChildTy
 /// in an [Align] widget. If the child might want to be bigger, consider
 /// wrapping it in a [SingleChildScrollView] or [OverflowBox].
 ///
+/// {@tool snippet}
```

Comment:
Optional suggestion: This will make the sample an interactive dartpad, which means you can resize the window and actually see this in action! :)

```suggestion
/// {@tool dartpad --template=stateless_widget_material}
```